src/tests/JIT/Methodical/MDArray/InnerProd/classarr.cs

```diff
@@ -123,11 +124,31 @@ public static int Main()
     {
         bool pass = false;
 
-        rand = new Random();
+        int seed = DEFAULT_SEED;
+
+        if (Environment.GetEnvironmentVariable("CORECLR_SEED") != null)
+        {
+            string CORECLR_SEED = Environment.GetEnvironmentVariable("CORECLR_SEED");
+
+            if (!int.TryParse(CORECLR_SEED, out seed))
+            {
+                if (string.Equals(CORECLR_SEED, "random", StringComparison.OrdinalIgnoreCase))
+                {
+                    seed = new Random().Next();
+                }
+                else
+                {
+                    seed = DEFAULT_SEED;
+                }
+            }
+        }
```

Comment:
Nit: here and in other places - I'd write it this way:
```CS
int seed = Environment.GetEnvironmentVariable("CORECLR_SEED") switch
{
    string seedStr when seedStr.Equals("random", StringComparison.OrdinalIgnoreCase) => new Random().Next(),
    string seedStr when int.TryParse(seedStr, out int envSeed) => envSeed,
    _ => DefaultSeed
};
```
In any case, `CORECLR_SEED` (if it is to stay) needs to be `camelCase`d as it is a local variable.

Reply:
I don't believe CORECRL_SEED can be camel cased as it's an environment variable in the CI pipeline.

Reply:
Of course, but I was only referring to the local variable: `string CORECLR_SEED = Environment.GetEnvironmentVariable("CORECLR_SEED");`.

Reply:
Made those suggestions and it looks much cleaner now, thank you.